index.html

```diff
@@ -5,8 +5,65 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
 </head>
+
```

Comment:
Looks like there's a few errors when run through a html validator. most of them are harmless, though. You can check out the error messages here: [https://validator.w3.org/](https://validator.w3.org/)